docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -60,44 +61,23 @@ class ComponentExample extends React.Component<ComponentExampleProps, ComponentE
   kebabExamplePath: string
   KnobsComponent: any
 
-  state = {
-    knobs: {},
-    themeName: 'teams',
-    componentVariables: {},
-    handleMouseLeave: _.noop,
-    handleMouseMove: _.noop,
-    showCode: false,
-    showRtl: false,
-    showTransparent: false,
-    showVariables: false,
-    isHovering: false,
-    copiedCode: false,
-  }
-
-  static contextTypes = {
-    onPassed: PropTypes.func,
-  }
+  constructor(props) {
+    super(props)
 
-  static propTypes = {
-    children: PropTypes.node,
-    description: PropTypes.node,
-    examplePath: PropTypes.string.isRequired,
-    history: PropTypes.object.isRequired,
-    location: PropTypes.object.isRequired,
-    match: PropTypes.object.isRequired,
-    title: PropTypes.node,
-    themeName: PropTypes.string,
-  }
-
-  componentWillMount() {
```

Comment:
https://reactjs.org/docs/react-component.html#unsafe_componentwillmount
This method is deprecated, switched to `constructor`